.bazelci/presubmit.yml

```diff
@@ -6,19 +6,29 @@ platforms:
   macos:
     build_targets:
       - "..."
+    test_targets:
+      - "//..."
```

Comment:
Why these exclusions?

Reply:
That's YAML's list syntax, the exclusion would be `- "-//..."` ;)

Reply:
well that's embarrassing